torch/csrc/jit/codegen/cuda/partition.cpp

```diff
@@ -17,6 +17,19 @@ const c10::DeviceIndex INVALID_INDEX = -2;
 
 namespace {
 
+bool getEnableComplexEnv() {
+  char* e = std::getenv("PYTORCH_NVFUSER_ENABLE_COMPLEX_PY");
```

Comment:
I'm gonna suggest that we follow the pattern here: #1666.

maybe we can rebase this on Naoya's PR or wait for that one to be merged first?

Reply:
Looks like the https://github.com/csarofeen/pytorch/pull/1666 assumes enabled by default? Here I want it to be disabled by default. Do you think I should add a similar mechanism `PYTORCH_NVFUSER_ENABLE`, or should I keep this flag as is?

Reply:
Adding a `PYTORCH_NVFUSER_ENABLE` sounds good to me~ :pray: